src/aiogram_dialog/api/entities/stack.py

```diff
@@ -43,6 +43,7 @@ class Stack:
     last_income_media_group_id: Optional[str] = field(
         compare=False, default=None,
     )
+    key: Optional[str] = field(compare=False, default=None)
```

Comment:
can't see if it is somehow used

Reply:
Sorry, "add key field" is unwanted commit that I made for project purpose. Only arguments typing commit matter